pills/05-functions-and-imports.xml

```diff
@@ -1,8 +1,348 @@
 <chapter xmlns="http://docbook.org/ns/docbook"
-      xmlns:xlink="http://www.w3.org/1999/xlink"
-      xmlns:xi="http://www.w3.org/2001/XInclude"
-      version="5.0"
-      xml:id="functions-and-imports">
+xmlns:xlink="http://www.w3.org/1999/xlink"
+xmlns:xi="http://www.w3.org/2001/XInclude" version="5.0"
+xml:id="functions-and-imports">
+
+      <title>functions and imports</title>
+
+<section>
+      <title>Functions</title>
+      <para>
+            Welcome to the fifth Nix pill. In the previous <link
+            linkend="basics-of-the-language">fourth pill</link> we touched the
+            Nix language for a moment. We introduced basic types and values of
+            the Nix language, and basic expressions such as
+            <code>if</code>, <code>with</code> and
+            <code>let</code>. I invite you to re-read about these
+            expressions and play with them in the repl.
+      </para>
+
+      <para>
+            Functions help to build reusable components in a big repository like
+            <link xlink:href="https://github.com/NixOS/nixpkgs/">nixpkgs</link>.
+            The Nix manual has a <link
+            xlink:href="https://nixos.org/nix/manual/#ss-functions">great
+            explanation of functions</link>. Let's go: pill on one hand, Nix
+            manual on the other hand.
+      </para>
+
+      <para>
+            I remind you how to enter the Nix environment: <code>source
+            ~/.nix-profile/etc/profile.d/nix.sh</code>
+      </para>
+
+</section>
+
+<section>
+      <title>Nameless and single parameter</title>
```

Comment:
To remain consistent with the rest of the doc, can you use 2 space indents, and keep sections indented one level in (as it is a child of the `chapter`)? I've created an editorconfig file to include the indentation information.